code/__DEFINES/span.dm

```diff
@@ -0,0 +1,179 @@
+// Sorted alphabetically
+#define span_abductor(str) ("<span class='abductor'>" + str + "</span>")
```

Comment:
inb4 maint comment, but this specific code got denied in the past. 

https://github.com/BeeStation/BeeStation-Hornet/pull/4971

I think the reasoning for the denial was stupid, considering most other modern codebases use it and have made peace with it, yet contribs here still have the write out the whole damn thing every time.

Just something to keep in mind.

Reply:
I see
I will replace what needs be if this file is unwelcome
didn't know it was a hot topic

Reply:
As said previously, I'm not as against them as I used to be but at the same time if we are doing this it shouldn't be here.

Reply:
I'll remove this then

Reply:
Fixed